src/vs/workbench/contrib/tags/electron-browser/workspaceTagsService.ts

```diff
@@ -468,10 +512,70 @@ export class WorkspaceTagsService implements IWorkspaceTagsService {
 					// Ignore errors when resolving file or parsing file contents
 				}
 			});
-			return Promise.all([...packageJsonPromises, ...requirementsTxtPromises, ...pipfilePromises]).then(() => tags);
+
+			const pomPromises = getFilePromises('pom.xml', this.fileService, this.textFileService, content => {
+				try {
+					let dependenciesContent;
+					while (dependenciesContent = dependenciesRegex.exec(content.value)) {
+						let dependencyContent;
```

Comment:
Isn't the following lines of code the same as calling processGradleDependencies?

Reply:
Maven differs a bit with gradle as I need to use different regex to get `groupId` and `artifactId` for maven but I could get both of them in one regex for maven

Reply:
okay